src/applications/ivc-champva/10-7959C/chapters/applicantInformation.js

```diff
@@ -22,7 +22,12 @@ fullNameMiddleInitialUI.middle['ui:title'] = 'Middle initial';
 
 export const applicantNameDobSchema = {
   uiSchema: {
-    ...titleUI('Beneficiary’s name'),
+    ...titleUI(
+      ({ formData }) =>
+        `${
+          formData.certifierRole === 'applicant' ? 'Your' : 'Beneficiary’s'
+        } name`,
+    ),
```

Comment:
Since we moved the "Who are you?" question up in the form flow, this can now dynamically display "Your name" or "Beneficiary's name" depending.